clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -91,11 +93,26 @@ def start_clipper(
             compability and preserve the expected behavior of the system.
         cache_size : int, optional
             The size of Clipper's prediction cache in bytes. Default cache size is 32 MiB.
-
+        config_file: str(optional)
+            File for the configuration of the application or model
         Raises
         ------
         :py:exc:`clipper.ClipperException`
         """
+        if(config_file is not None):
+           with open(config_file, "r") as stream:
+                config = yaml.load(stream)
+                for key in config:
+                    name = config[key]["name"]
+                    version = config[key]["version"]
+                    input_type = config[key]["input_type"]
+                    obj_type = config[key]["obj_type"]
+                    config_image = config[key]["image"]
+                    slo = config[key]["slo"]
+                    if(obj_type == "application"):
+                        self.register_application(name, input_type, "DEFAULT", slo)
```

Comment:
On second thought, let's force the user to specify a "default_value" parameter in the **application** configuration (this is not needed for models) because the "default_value" python parameter does not have a default value.